### ISSUE TITLE
Booleans are treated as numeric instead of categorical

### ISSUE THREAD

**Issue by LeTristanB** [JosPolfliet/pandas-profiling#25]
A column containing `True` or `False` will be summarized like a numeric column and trigger numeric warnings such as `was probed successfully has 651907 / 94.4% zeros` . Here's an example http://i.imgur.com/OaCPJ86.png . Boolean should be summarized like a categorial variable.

**Comment by JosPolfliet** [comment]
You're right, it does not make sense any other way. Thanks for catching. Feel free to update the code and send a pull request.

**Comment by romainx** [comment]
Hello,

I've implemented a **new Boolean variable type**, see #64.
The implementation is close the categorical variable type, but I think it's useful to make the distinction between the two. For example, the **mean** stats is computed for boolean variables because it makes sense. 

```Python
df = pd.DataFrame({'Is it correct?':[True, True, False, True, True]})
pandas_profiling.ProfileReport(df)
````
<img width="966" alt="boolean" src="https://user-images.githubusercontent.com/7131913/32264069-2a10e0f0-bedd-11e7-8135-f94dd0c2fe01.png">
